scale-tests/deploy-dispatchers.py

```diff
@@ -63,6 +63,8 @@
 
 log = logging.getLogger(__name__)
 
+MAX_THREADPOOL_WORKERS = 50
```

Comment:
Should this be configurable, either through an envvar or a command-line option?

Reply:
I also don't know if it's required to be set here. We could just use a local variable were we instantiate the `ThreadPoolExecutors`.

Reply:
> Should this be configurable, either through an envvar or a command-line option?

I thought about doing that but decided otherwise. Do you think we should?

Reply:
> I also don't know if it's required to be set here. We could just use a local variable were we instantiate the ThreadPoolExecutors.

That's just my personal preference for setting "magic values" as really visible constants instead of inside functions. If you feel strongly about it I can change it.

Reply:
Let's make it configurable if and when we require it. I'm ok with it as it is.